### PR TITLE
Code of conduct

### PR DESCRIPTION
**Pull request by opdavies**
A couple of questions for you, @DaveLiddament:

- I've made the website info data available to all templates. Is this the correct place to do it?
- Are we going to introduce forms like the "Report a problem" one currently on http://phpsw.uk/code-of-conduct, or were you planning on a different approach? This also applies to the "Submit a talk" and "Interested in becoming a sponsor?" forms elsewhere.

### CODE REVIEW

File: app/src/WebsiteGenerator/TemplateRenderer/TemplateRenderer.php [hunk 1]
```diff
@@ -59,6 +69,8 @@ public function render(string $filename, string $templateName, array $data)
 
         $data['page']['template'] = $templateName;
 
+        $this->twig->addGlobal('info', $this->websiteInfoRepository->getWebsiteInfo());
```
Comment (DaveLiddament) [inline app/src/WebsiteGenerator/TemplateRenderer/TemplateRenderer.php:72]: This is a question more than a comment. 

What does the addGlobal mean and how is it different to just adding the the array we hand over to twig?

File: app/resources/views/codeOfConduct.html.twig [hunk 2]
```diff
@@ -1,6 +1,65 @@
...
+        <p>tl;dr: Don’t be a Jerk</p>
+
+        <h2>Need help?</h2>
+
+        <p>You can contact the organisers at info@phpsw.org.uk, send a direct message to @phpsw on Twitter or speak to an organiser, in person, at an event.</p>
+
+        <p>The current organisers are:</p>
+
+        <ul>
+            <li>Lee Stone (@leesto)</li>
```
Comment (DaveLiddament) [inline app/resources/views/codeOfConduct.html.twig]: Can we replace this by looping through organisers?

Comment (DaveLiddament) [reply app/resources/views/codeOfConduct.html.twig]: Ah, you have in a later commit. My bad!

File: app/src/WebsiteGenerator/TemplateRenderer/TemplateRenderer.php (hunk 1 above)
Comment (opdavies) [reply app/src/WebsiteGenerator/TemplateRenderer/TemplateRenderer.php:72]: In this case, it's no different. But it does mean that the info can be used in other places where it's needed like in base.html.twig and the navigation and footer partials.

Comment (DaveLiddament) [reply app/src/WebsiteGenerator/TemplateRenderer/TemplateRenderer.php:72]: Great. In that case yes I think that's exactly what we should be doing.

**Review by DaveLiddament — APPROVED** [review]

**Pull request merged**